srv/cmd/main.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main implements a simple HTTP/JSON REST example.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/abcxyz/abc-updater/srv/pkg"
+
+	"github.com/abcxyz/pkg/logging"
+	"github.com/abcxyz/pkg/renderer"
+	"github.com/abcxyz/pkg/serving"
+)
+
+const defaultPort = "8080"
+
+var port = flag.String("port", defaultPort, "Specifies server port to listen on.")
+
+// TODO: figure out how to make modules so this doesn't get re-defined multiple places
+type SendMetricRequest struct {
+	// The ID of the application to check.
+	AppID string `json:"appId"`
```

Comment:
I think we tend to use snake case for json. `app_id`

Reply:
There is unfortunately already part of the client using camelCase (for updater) which has been merged into `abc`. Would it make sense to have split conventions on the same app?

I have https://github.com/abcxyz/readability/issues/28 open but it never got resolved (we had a lot of debate in team chat with no resolution). I originally wanted to make the client support snake or camel case, but unfortunately golang can't easily support multiple bindings for the same field.

Reply:
Thanks. We can treat that separately.